python-avd/pyavd/_eos_designs/structured_config/core_interfaces_and_l3_edge/utils.py

```diff
@@ -302,18 +302,11 @@ def _get_ethernet_cfg(self: AvdStructuredConfigCoreInterfacesAndL3Edge, p2p_link
         """
         ethernet_cfg = {"speed": p2p_link.get("speed")}
 
-        if get(p2p_link, "ptp.enabled") is not True:
+        if get(p2p_link, "port_channel") is not None:
```

Comment:
Instead of this I think the PTP should be moved to some other get_xxxx function, only called when it is not a port-channel. Please take a step back and analyze the chain of functions called and see where it fits. Thanks.